README.md

```diff
@@ -15,7 +15,7 @@ const ecc = require('eosjs-ecc')
 
 # Include
 
--   Install with: `npm install eosjs-ecc`
+-   Install with: `yarn add eosjs-ecc`
```

Comment:
why this change?  Also below you tell them to use yarn . Can't a user use either yarn OR npm, also I thought we were sticking with yarn for OSS.

Reply:
Yes, a user can use either but we are making the change over to yarn in all of our documentation.